packages/store/addon/-private/system/record-data-for.ts

```diff
@@ -16,14 +19,28 @@ interface InternalModel {
   _recordData: RecordData;
 }
 
+const IdentifierCache = new WeakMap<StableRecordIdentifier, RecordData>();
+
+export function setRecordDataFor(identifier: StableRecordIdentifier, recordData: RecordData) {
```

Comment:
Can we say identifier in this method name somewhere please, especially as recordDataFor has multiple signatures